.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,51 @@
+name: Docker
+
+on:
+  push:
+    branches:
+      - master
+      - Dockerfile-use-installer
+  pull_request:
+    branches:
+      - master
+      - Dockerfile-use-installer
+  workflow_dispatch:
+
+env:
+  IMAGE_NAME: fomu-toolchain
+
+jobs:
+  push:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push'
+
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Please, use `fetch-depth` (as in https://github.com/im-tomu/fomu-toolchain/blob/master/.github/workflows/update.yml#L23), for ensuring that tags are retrieved.